docs/release-notes/snapcraft-8-11.rst

```diff
@@ -0,0 +1,49 @@
+.. _release-8.11:
+
+Snapcraft 8.11 release notes
+============================
+
+4 August 2025
+
+Learn about the new features, changes, and fixes introduced in Snapcraft 8.11.
+
+
+Requirements and compatibility
+------------------------------
+See :ref:`reference-system-requirements` for information on the minimum hardware and
+installed software.
+
+
+Backwards-incompatible changes
+------------------------------
+
+The following changes are incompatible with previous versions of Snapcraft.
+
+
+Snap file name changes
+~~~~~~~~~~~~~~~~~~~~~~
+
+The file name for core24 snaps now use the platform name instead of
+the build-for architecture.
```

Comment:
```suggestion
The file names for core24 snaps are now derived from the name of the entry in the 
``platforms`` key instead of the value of its ``build-for`` key.
```